### PR TITLE
Quick Start Docs Callout

### PR DESCRIPTION
**Pull request by spbyrne**
The text will need some tweaking. Maybe there's a way to do this with a shortcode instead of inline HTML?

Screen Shot 2021-05-25 at 5 09 53 PM

Preview: https://tinacms-site-next-git-quickstart-callout-tinacms.vercel.app/docs/

### CODE REVIEW

**Comment by jeffsee55** [comment]
Oh that's muuuch better than what I had in mind...

File: content/docs/index.md [hunk 1]
```diff
@@ -9,6 +9,15 @@ last_edited: '2020-11-26T14:37:54.481Z'
 - The **TinaCMS toolkit**, a suite of NPM packages that enable developers to quickly create a custom content management frontend for a website.
 - **Tina Cloud**, a hosted service designed to make Git-based content management more powerful and accessible for cross-functional teams.
 
+<div class="callout">
+<img className="learnImage" src="../img/tina-laptop.png" alt="" />
+  <div>
+  <h3>Ready to get started?</h3>
+  <p>Read our getting started guide to get set up in no time.</P>
```
Comment (DirtyF) [inline content/docs/index.md:16]:
```suggestion
  <p>Get a Tina running website in no time!</p>
```

Comment (spbyrne) [reply content/docs/index.md:16]: @DirtyF  How about "Get a website running with Tina Cloud in no time!"?

Comment (DirtyF) [reply content/docs/index.md:16]: even better!

**Comment by DirtyF** [comment]
I guess MDX would be the way to go if we want to use components in Markdown, but we don't support that do we?

File: content/docs/index.md (hunk 1 above)
Comment (spbyrne) [reply content/docs/index.md]: Done.

**Comment by dwalkr** [comment]
Looks great to me. FYI this website does support shortcodes via `remark-shortcode` https://github.com/tinacms/tinacms.org/blob/master/README.md#shortcodes

however, they aren't supported by the WYSIWYG editor.

**Review by dwalkr — APPROVED** [review]

**Pull request merged**